server/app.ts

```diff
@@ -34,6 +35,11 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebSession())
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
+
+  if (config.featureToggles.frontendComponentsApiToggleEnabled) {
+    setUpEnvironmentName(app)
```

Comment:
This is for the Header fallback https://github.com/ministryofjustice/hmpps-micro-frontend-components/blob/main/readme/incorporating.md#fallbacks-services-with-prison-only-users

Reply:
This needs to become before the `setupNunjucks` otherwise I would have grouped it with the other `if` statement block

Reply:
I reckon we can do with without the if block, as its only setting a couple of locals - its not actually functional per-se, so probably doesn't matter if we do it even if the feature toggle is disabled?

Also, I'm not sure a function in the middleware folder is adding much (especially as I don't think its strictly speaking a middleware function?

Could we replace what you have here:
```
  if (config.featureToggles.frontendComponentsApiToggleEnabled) {
    setUpEnvironmentName(app)
  }
```

with just:
```
  app.locals.environmentName = config.environmentName
  app.locals.environmentNameColour = config.environmentName === 'PRE-PRODUCTION' ? 'govuk-tag--green' : ''
```  

And then delete the middleware function ?
What do you think?

Reply:
Yep that makes sense, will just have to remember this if they make any further changes to the fallback!